### DISCUSSION TITLE
Install webui with cuda

### DISCUSSION THREAD

**Discussion by pandaymx** [open-webui/open-webui#4060] [General]
# Bug Report

## Description

I run `docker run -d -p 3000:8080 --gpus all --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:cuda`

but python has a error.

```
RuntimeError: Unexpected error from cudaGetDeviceCount(). Did you run some cuda functions before calling NumCudaDevices() that might have already set an error? Error 500: named symbol not found
```
## Environment

- **Open WebUI Version:** default
- **Ollama (if applicable):**  0.2.8

- **Operating System:** Windows 11

**Comment by DashBing** [comment]
Did you install Conda?
